src/main/java/com/example/repositories/PhoneRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.example.repositories;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.querydsl.QueryDslPredicateExecutor;
+
+import com.example.domain.Employee;
+import com.example.domain.Phone;
+
+public interface PhoneRepository extends JpaRepository<Phone, Long>, QueryDslPredicateExecutor<Phone> {
+	
+	List<Phone> findByOwner(Employee owner);

```

Comment:
What do you mean?

findByOwner is the simplest query supported by Spring Data.
There's no need for querydsl at all for this one line of code.